sw/device/lib/base/memory_unittest.cc

```diff
@@ -71,12 +86,12 @@ class MemSetTest : public ::testing::TestWithParam<decltype(&ot_memset)> {};
 class MemChrTest : public ::testing::TestWithParam<decltype(&ot_memchr)> {};
 
 INSTANTIATE_TEST_SUITE_P(MemCpy, MemCpyTest,
-                         ::testing::Values(ot_memcpy, __builtin_memcpy));
+                         ::testing::Values(ot_memcpy, builtin_memcpy_wrapper));
```

Comment:
Interesting, do you remember what the error was?

Reply:
Yes, it was `builtin functions must be directly called`

Reply:
relevant [thread](https://gcc.gnu.org/legacy-ml/gcc-bugs/2015-06/msg01141.html) (which is pretty old)